statsmodels/graphics/tests/test_tsaplots.py

```diff
@@ -51,8 +51,8 @@ def test_plot_month():
     dta = sm.datasets.elnino.load_pandas().data
     dta['YEAR'] = dta.YEAR.astype(int).apply(str)
     dta = dta.set_index('YEAR').T.unstack()
-    dates = lmap(lambda x : pd.datetools.parse('1 '+' '.join(x)),
```

Comment:
All I'm trying to do is fix old code here. This worked until pandas 0.16.0 and now doesn't:

    In [2]: pd.__version__
    Out[2]: u'0.17.0+73.g5d953e3'

    In [3]: pd.datetools
    Out[3]: <module 'pandas.core.datetools' from '/home/rgommers/.local/lib/python2.7/site-packages/pandas/core/datetools.pyc'>

    In [4]: pd.datetools.parse
    ---------------------------------------------------------------------------
    AttributeError                            Traceback (most recent call last)
    <ipython-input-4-e5633a1e4d6d> in <module>()
    ----> 1 pd.datetools.parse

    AttributeError: 'module' object has no attribute 'parse'

Could be that the old code was just using something that looked public (an imported ``parse`` function).